### PR TITLE
Fix missing host in email title

### PR DESCRIPTION
**Pull request by coorasse**
image

### CODE REVIEW

**Review by martinemde — APPROVED** [review]
Nice! Thank you for noticing a problem and actually figuring out the fix for it.

**Comment by simi** [comment]
Thanks for the fix. Would you mind to add simple test to prevent this from happening in the future?

Sadly mailers are not well covered in testing (yet), but something simple like following in `test/unit/mailer_test.rb` would be good start.

```ruby
  context "email reset update" do
    setup do
      @user = create(:user)
    end

    should "include host in subject" do
      email = Mailer.email_reset_update(@user)

      assert_emails 1 do
        email.deliver_now
      end

      assert_includes email.subject, Gemcutter::HOST_DISPLAY
    end
  end
```

**Comment by martinemde** [comment]
Thank you for adding the test.

File: test/mailers/mailer_test.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+require 'test_helper'
```
Comment (simi) [inline test/mailers/mailer_test.rb:1]: Is there any reason to keep this in separate file? There are already tests in `test/unit/mailer_test.rb`. I'm happy to move those to `test/mailers` if that's the default path for generating mailer tests in Rails today (I can do this in separate PR on my own).

Comment (simi) [inline test/mailers/mailer_test.rb:1]: to make CI happy this change is needed

```suggestion
require "test_helper"
```

**Comment by simi** [comment]
Is there any reason to keep test in separate file? There are already tests in `test/unit/mailer_test.rb`. I'm happy to move those to `test/mailers` if that's the default path for generating mailer tests in Rails today (I can do this in separate PR on my own).

**Comment by coorasse** [comment]
To be honest, I went straight to the mailers folder and I did not check the unit folder. Yes, if you generate a mailer with `bin/rails g mailer whatever` the tests are created under `test/mailers`, so you can move everything there 👍

**Pull request merged**